src/slack/events/message.ts

```diff
@@ -7,9 +7,10 @@ import logger from '../../logger';
 function register(bolt: App): void {
   bolt.message(async ({ say }) => {
     // TODO: get context (toggles), use to generate specific dashboard blocks
-    const teamRegistrationToggle = (await Config.findOne({ key: 'teamRegistrationActive' })) || { key: 'teamRegistrationActive', value: 'false' };
+    const teamRegistrationActiveKey = 'teamRegistrationActive';
+    const teamRegistrationActive = await Config.findToggleForKey(teamRegistrationActiveKey);
     const context = {
-      [teamRegistrationToggle.key]: teamRegistrationToggle.value === 'true',
+      teamRegistrationActiveKey: teamRegistrationActive,
```

Comment:
I assume there should be brackets around this? Or should the key actually be `teamRegistrationActiveKey`?

```suggestion
      [teamRegistrationActiveKey]: teamRegistrationActive,
```